public/app/core/components/sidemenu/BottomNavLinks.test.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import BottomNavLinks from './BottomNavLinks';
+import appEvents from '../../app_events';
+
+jest.mock('../../app_events', () => ({
+  emit: jest.fn(),
+}));
+
+const setup = (propOverrides?: object) => {
+  const props = Object.assign(
+    {
+      link: {},
+      user: {
```

Comment:
There's a type error:
```
Type '{ orgName: string; orgCount: number; }' is not assignable to type 'User'.
  Property 'isGrafanaAdmin' is missing in type '{ orgName: string; orgCount: number; }'.
```
Seems, you can use `Partial` to solve this.

Reply:
Solved by specifying full set of properties on mock